Wikipedia/Code/ReadingThemesControlsViewController.xib

```diff
@@ -256,6 +261,38 @@
                                         <constraint firstAttribute="trailing" secondItem="56n-xZ-69v" secondAttribute="trailing" constant="17" id="m0h-MT-JGM"/>
                                     </constraints>
                                 </view>
+                                <view hidden="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Avb-i2-HSA" userLabel="Separator 3">
+                                    <rect key="frame" x="0.0" y="240" width="290" height="0.0"/>
+                                    <color key="backgroundColor" red="0.95294117649999999" green="0.92549019610000005" blue="0.94117647059999998" alpha="1" colorSpace="calibratedRGB"/>
+                                    <constraints>
+                                        <constraint firstAttribute="height" priority="999" constant="1" id="1me-Am-mAC"/>
+                                    </constraints>
+                                </view>
+                                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Kkd-Mq-to7">
+                                    <rect key="frame" x="0.0" y="240" width="290" height="52"/>
+                                    <subviews>
+                                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Syntax Highlighting" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="uMd-Oo-cjP">
+                                            <rect key="frame" x="10" y="15" width="211" height="24"/>
+                                            <fontDescription key="fontDescription" style="UICTFontTextStyleTitle2"/>
```

Comment:
Note I used the system Title 2 here, but let me know if there's a preferred way to do text styles.

Reply:
You could override `traitCollectionDidChange` and then use `wmf_font`:

```
    open override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
        super.traitCollectionDidChange(previousTraitCollection)
        syntaxHighlightingLabel.font = UIFont.wmf_font(.[font type], compatibleWithTraitCollection: traitCollection)
    }
```

Reply:
👍 done!